TWLight/graphs/helpers.py

```diff
@@ -49,9 +50,14 @@ def get_earliest_creation_date(queryset):
     # Some imported applications had no date and were given
     # creation dates of Jan 1, 1970. This screws up the graphs.
     if queryset:
-        earliest_date = queryset.exclude(
-            date_created=datetime.date(1970,1,1)).earliest(
-                'date_created').date_created
+        try:
```

Comment:
Why is this now wrapped in a try block? Comments here would be helpful.

Reply:
This feels confusing - we're using this function to operate on both authorization querysets and application querysets, such that we have to rely on `try` to circumvent different field names. Should these be split out into separate functions?

Reply:
Alternatively, should Authorizations `date_authorized` field be renamed `date_created` so that we're using conventions for this general metadata?